src/components/PhoneInput/PhoneInput.js

```diff
@@ -0,0 +1,86 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import { uuid4 } from '../../utils/utils';
+import { LABEL_CLASSES } from '../../constants/classes';
+
+/**
+ * @uxpincomponent
+ */
+
+export default class PhoneInput extends React.Component {
+  key = 0;
+
+  constructor(props) {
+    super(props);
+    this.state = { id: uuid4() };
+  }
+
+  render() {
+    this.key += 1;
+
+    return (
+      <div
+        key={this.key}
+        style={{
+          width: `${this.props.width ? `${this.props.width}px` : ''}`,
+        }}>
+        <chi-label for={`phone-input-${this.state.id}`}>{this.props.label}</chi-label>
+        <chi-phone-input
+          default-country={this.props.country}
+          disabled={this.props.disabled}
+          id={`phone-input-${this.state.id}`}
+          placeholder={this.props.placeholder}
+          size={this.props.size}
+          state={this.props.state}
+          value={this.props.value}>
+        </chi-phone-input>
+        {this.props.state && (
+          <div className={`${LABEL_CLASSES.LABEL} -status -${this.props.state}`}>
+            {this.props.stateLabel}
+          </div>
+        )}
+      </div>
+    );
+  }
+}
+
+PhoneInput.propTypes = {
+  disabled: PropTypes.bool,
+  country: PropTypes.oneOf([
```

Comment:
As agreed with Tom, let's list only the main countries here. Also, we might need to map the full name and abbreviation of the countries like:
"Italy" --> "IT"
So that the designers see the actual name, not the abbreviation.